### PR TITLE
feat(66): add integration with firebase for notification

### PR DESCRIPTION
**Pull request by 0to1to0to1**

### CODE REVIEW

File: packages/mobile/src/components/Button/Button.tsx [hunk 1]
```diff
@@ -22,10 +22,11 @@ export const CustomButton = ({
   variant,
   title,
   icon,
+  // testIcon,
   ...buttonProps
 }: CustomButtonProps) => {
   const isPrimary = variant === 'primary' ? true : false;
-  const iconSource = icon && ICONS[icon];
+  // const iconSource = icon && ICONS[icon];
```
Comment (dubbha) [inline packages/mobile/src/components/Button/Button.tsx]: Do you expect to still need these in future?

File: packages/mobile/src/pages/ScreensAuthorization/Authorization/Authorization.tsx [hunk 2]
```diff
@@ -7,6 +7,8 @@ import { CustomButton } from '../../../components/Button';
 import { Divider } from '../../../components/Divider';
 import styles from './styles';
 const autorizationBgImg = require('./../../../assets/img/bg-home.jpg');
+const iconSrcGoogle = './../../../assets/icons/google.png';
+const iconSrcFb = './../../../assets/icons/fb.png';
```
Comment (dubbha) [inline packages/mobile/src/pages/ScreensAuthorization/Authorization/Authorization.tsx]: Why not requiring them here as you do `authorizationBgImg`?

File: packages/mobile/ios/croton/AppDelegate.m [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+- (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
+{
+#if DEBUG
+  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"packages/mobile/index" fallbackResource:nil];
+#else
+  return [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
+#endif
+}
+
+@end
```
Comment (dubbha) [inline packages/mobile/ios/croton/AppDelegate.m:89]: Give us an overview of this magic on our next syncup :)

File: packages/mobile/android/app/BUCK [hunk 4]
```diff
@@ -35,12 +35,12 @@ android_library(
 
 android_build_config(
     name = "build_config",
-    package = "com.myprojectname",
+    package = "com.croton",
```
Comment (dubbha) [inline packages/mobile/android/app/BUCK:38]: That's a better name!

**Review by dubbha — APPROVED** [review]
See my comments, fine otherwise

File: packages/mobile/src/components/Button/Button.tsx (hunk 1 above)
Comment (artbitbyte) [reply packages/mobile/src/components/Button/Button.tsx]: I didn't remove some comment that can helps make changes according to my inner todo list.
But I will remove this, thanks :)

File: packages/mobile/android/app/BUCK (hunk 4 above)
Comment (artbitbyte) [reply packages/mobile/android/app/BUCK:38]: Yep :)

File: packages/mobile/ios/croton/AppDelegate.m (hunk 3 above)
Comment (artbitbyte) [reply packages/mobile/ios/croton/AppDelegate.m:89]: :) This is ease recommendation for notify setup on iOS

File: packages/mobile/src/pages/ScreensAuthorization/Authorization/Authorization.tsx (hunk 2 above)
Comment (artbitbyte) [reply packages/mobile/src/pages/ScreensAuthorization/Authorization/Authorization.tsx]: I will fix this, thanks.

File: packages/mobile/src/store/shelves/sagas/shelfFlowerAdd.saga.ts [hunk 5]
```diff
@@ -16,7 +16,6 @@ function* handle(action: ShelfFlowerAdd) {
       body: action.payload,
     });
 
-    console.dir(result);
     if (result.status === 403) {
```
Comment (dartwlad) [inline packages/mobile/src/store/shelves/sagas/shelfFlowerAdd.saga.ts]: I would suggest moving status codes to some reuseable constants

**Review by dartwlad — APPROVED** [review]

File: packages/mobile/src/store/shelves/sagas/shelfFlowerAdd.saga.ts (hunk 5 above)
Comment (artbitbyte) [reply packages/mobile/src/store/shelves/sagas/shelfFlowerAdd.saga.ts]: Thanks, I will fix it.

**Pull request merged**